src/Symfony/Bundle/FrameworkBundle/Resources/config/translation.php

```diff
@@ -114,6 +114,10 @@
         ->set('translation.dumper.xliff', XliffFileDumper::class)
             ->tag('translation.dumper', ['alias' => 'xlf'])
 
+        ->set('translation.dumper.xliff', XliffFileDumper::class)
```

Comment:
I guess we have to use a different id here.

Reply:
Ah yes, i forgot to mention it. Since the dumper with the suffix `xlf` has already the id `translation.dumper.xliff` i have some problems finding a suitable service name. `translation.dumper.xlf` would be confusing i think.

Reply:
`translation.dumper.xliff.xliff`?

Reply:
I would be fine with it. Do you think it we should later on change the service names to respectively `translation.dumper.xlf` and `translation.dumper.xliff`?

Reply:
to translation.dumper.xliff.xliff and translation.dumper.xliff.xlf maybe, but there's nothing urgent about that